examples/vault-consul-ami/vault-consul.json

```diff
@@ -113,9 +113,13 @@
   },{
     "type": "shell",
     "inline": [
-      "sudo yum install -y git"
+      "sudo yum install -y git",
+      "if [ '{{user `install_auth_signing_script`}}' = 'true' ]; then",
```

Comment:
Please use `[[ ]]` and `==`.

Reply:
I initially changed all three occurrences to use `[[ ]]` and `==`, but discovered that this failed when building the Ubuntu image, getting an error (`[[: not found`).  I was able to work around this by specifying `#!/bin/bash -e` as the `inline_shebang` parameter.  This seems to only be necessary when building the Ubuntu image, so I did not specify it for the Amazon Linux 2 image.